crates/rpc/rpc/src/eth/core.rs

```diff
@@ -444,8 +444,9 @@ mod tests {
             last_header = Some(header.clone());
             parent_hash = hash;
 
-            let mut transactions = vec![];
-            for _ in 0..100 {
+            let transactions_len: usize = 100;
```

Comment:
better of as a const, even if it stays within this scope, but best give it a name that makes sense when it's moved out of this scope to the top of the module